common/constants/autocomplete.ts

```diff
@@ -6,6 +6,8 @@
 
 import { BaseItem } from '@algolia/autocomplete-core';
 
+export const textAreaMaxLength = 16384;
```

Comment:
Just curious: where is this limit from? I don't see 2^14 often.

Reply:
arbitrary amount where I looked at how long certain sql queries could be, i made a value judgement. we can always change it later, though